spec/lib/pdf_fill/filler_spec.rb

```diff
@@ -31,11 +31,7 @@
       it 'generates extras and combine the files' do
         file_path = 'tmp/pdfs/file_path_final.pdf'
         expect(extras_generator).to receive(:generate).once.and_return('extras.pdf')
-        expect(described_class::PDF_FORMS).to receive(:cat).once.with(
-          old_file_path,
-          'extras.pdf',
-          file_path
-        )
+        expect(described_class).to receive(:merge_pdfs).once.with(old_file_path, 'extras.pdf', file_path)
```

Comment:
@jerekshoe Do you mind adding a direct unit tests for the new merge_pdfs method?